src/sidebar/test/oauth-client-test.js

```diff
@@ -23,7 +23,7 @@ const fixtures = {
   },
 };
 
-describe('sidebar.util.oauth-client', () => {
+describe('sidebar/oauth-client', () => {
```

Comment:
The `OAuthClient` class is something that I think is actually potentially quite useful outside of the client. It is a generic authorization client for any OAuth service that supports the "Web Message" response mode (https://tools.ietf.org/html/draft-sakimura-oauth-wmrm-00) and could be extended to work with standard redirects in future, which would make it useful for any oauth service.